project/src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,205 @@
+package com.google.sps.data;
+
+import java.lang.StringBuilder;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Random; 
+import java.util.Set;
+
+/** Class containing Event object 
+Setters for variables that user can change about event
+*/
+public final class Event {
+  private final int eventId = 1000000;
+  private String name;
+  private String description;
+  private Set<String> labels;
+  private String location;
+  private LocalDate date;
+  private Set<VolunteeringOpportunity> opportunities;
+  private Set<User> attendees;
+  private User host;
+
+  @Override
+  public String toString() { 
```

Comment:
Just curious, why you need to override this?

Reply:
It's just for future debugging on our side. With the regular toString(), we would get eg. Event@19821f printed, but with a toString() function we can see all the fields/info about the Event.